add_custom_view_and_activity_example/src/main/java/com/videoeditorsdk/android/app/custom/MyAdsView.kt

```diff
@@ -0,0 +1,36 @@
+package com.videoeditorsdk.android.app.custom
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.Toast
+import androidx.annotation.MainThread
+import ly.img.android.pesdk.backend.views.abstracts.ImgLyUIRelativeContainer
+import com.videoeditorsdk.android.app.R
+import ly.img.android.pesdk.annotations.OnEvent
+import ly.img.android.pesdk.ui.model.state.UiStateMenu
+import ly.img.android.pesdk.ui.panels.MenuToolPanel
+
+class MyAdsView @JvmOverloads constructor(
+    context: Context, attrs: AttributeSet? = null
+) : ImgLyUIRelativeContainer(context, attrs), View.OnClickListener {
```

Comment:
Would be good to leave a comment here that there are other options like `ImgLyUIFrameContainer` and `ImgLyUILinearContainer` available which abstracts away the event listener setup.